src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -311,8 +314,12 @@ export const ReviewCollapsible = ({
       );
     }
     if (reviewAction === ReviewAction.UPDATE || reviewAction === ReviewAction.NAME_CHANGE) {
-      const oldValue = reviewLevel.historyData.oldState as string;
-      const newValue = reviewLevel.historyData.newState as string;
+      let oldValue = reviewLevel.historyData.oldState as string;
+      let newValue = reviewLevel.historyData.newState as string;
+      if (reviewLevel.valuePath.includes('treatments')) {
+        oldValue = getTxName(drugList, oldValue);
```

Comment:
Let's say the new and old values are actually not the treatment name but the treatment level. In this case it looks like, `getTxName` will just do nothing. It does seem a bit misleading. Maybe we can use a regex here to say the valuePath ends with `treatments/\d`?